syntax/std_os_wasm.go

```diff
@@ -0,0 +1,27 @@
+// +build wasm
```

Comment:
Could we weave `ctx` throughout the API and pass an `afero.Fs` wrapping a JavaScript object? Raise an issue.